app/Http/Controllers/ClassController.php

```diff
@@ -33,13 +44,19 @@ public function store(Request $request)
     {
         $validatedData = $request->validate([
             'className' => 'required|string|max:50',
+            'image' => 'nullable|image|mimes:jpeg,png,jpg,gif,svg|max:2048',
             'capacity' => 'required|integer',
             'price' => 'required|decimal:0,3',
             'timeFrom' => 'required',
             'timeTo' => 'required|after:timeFrom',
         ]);
         // dd($request->all());
-       $validatedData['isFulled']=isset($request->isFulled);
+        $validatedData['isFulled']=isset($request->isFulled);
+        $image_name=(empty($request->image ))?"null":$this->upload_image($request->image);
```

Comment:
my mentor recommends to me not to use  `empty` function, he claims that it has a bad internal implementation. use instead `isset()` or `$request->hasFile()`

Reply:
Thank you,  I'll  remember it. 🤍